daemon/api.go

```diff
@@ -569,6 +570,31 @@ func getOpInfo(c *Command, r *http.Request) Response {
 	return SyncResponse(task.Map(route))
 }
 
+func postOp(c *Command, r *http.Request) Response {
+	route := c.d.router.Get(c.Path)
+	if route == nil {
+		return InternalError(nil, "router can't find route for operation")
+	}
+
+	id := muxVars(r)["uuid"]
+	task := c.d.GetTask(id)
+	if task == nil {
+		return NotFound
+	}
+
+	decoder := json.NewDecoder(r.Body)
+	var b bool
```

Comment:
Why a bool?  Won't that create compatibility problems if we decide to actually use that value for anything else later?